Formula/universal-ctags.rb

```diff
@@ -0,0 +1,57 @@
+class UniversalCtags < Formula
+  desc "Maintained ctags implementation"
+  homepage "https://github.com/universal-ctags/ctags"
+  url "https://github.com/universal-ctags/ctags/archive/refs/tags/p5.9.20210523.0.tar.gz"
+  sha256 "00e5493454c7c013b996a24019b8f8cbd75968e1993fb56f7e2737b7309cc684"
+  license "GPL-2.0-only"
+  head "https://github.com/universal-ctags/ctags.git"
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "docutils" => :build
+  depends_on "pkg-config" => :build
+  depends_on "jansson"
+  depends_on "libyaml"
+  uses_from_macos "libxml2"
+  conflicts_with "ctags", because: "this formula installs the same executable as the ctags formula"
+
+  def install
+    opts = []
+    system "./autogen.sh"
+    system "./configure", "--prefix=#{prefix}", *opts
+    system "make"
+    system "make", "install"
+  end
+
+  def caveats
+    <<~EOS
+      Under some circumstances, emacs and ctags can conflict. By default,
+      emacs provides an executable `ctags` that would conflict with the
+      executable of the same name that ctags provides. To prevent this,
+      Homebrew removes the emacs `ctags` and its manpage before linking.
+      However, if you install emacs with the `--keep-ctags` option, then
+      the `ctags` emacs provides will not be removed. In that case, you
+      won't be able to install ctags successfully. It will build but not
+      link.
+    EOS
+  end
+
+  test do
+    (testpath/"test.c").write <<~EOS
+      #include <stdio.h>
+      #include <stdlib.h>
+
+      void func()
+      {
+        printf("Hello World!");
+      }
+
+      int main()
+      {
+        func();
+        return 0;
+      }
+    EOS
+    system "#{bin}/ctags", "-R", "."
```

Comment:
```suggestion
    system bin/"ctags", "-R", "."
```
We don't need interpolation here.